tasks/main.yml

```diff
@@ -22,6 +22,37 @@
   ignore_errors: "{{ ansible_check_mode }}"
   tags: install
 
+- name: Create temporary inventory directory
```

Comment:
This role is designed to install netdata on non openio nodes which don't have an inventory.yml file.

This condition is thus necessary (which is why it was in a block):
`when: inventory_hostname in openio_netdata_oio_hosts`